src/main/java/org/pesmypetcare/webservice/service/usermanager/UserService.java

```diff
@@ -21,7 +21,7 @@
      * @throws DatabaseAccessException If an error occurs when accessing the database
      * @throws FirebaseAuthException If an error occurs when retrieving the data
      */
-    void createUser(String uid, UserEntity userEntity) throws DatabaseAccessException, FirebaseAuthException;
+    void createUser(String uid, UserEntity userEntity) throws DatabaseAccessException, FirebaseAuthException, DocumentException;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 128).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5641243)